extensions.json

```diff
@@ -163,6 +163,13 @@
       "repository": "https://github.com/ChristianKohler/PathIntellisense",
       "version": "2.2.1"
     },
+    {
+      "id": "christianvoigt.argdown-vscode",
+      "repository": "https://github.com/christianvoigt/argdown",
+      "version": "1.5.0",
+      "checkout": "argdown-vscode@1.5.0 && rm -f .yarnrc",
```

Comment:
> I'm not sure that hack with `rm -f .yarnrc` in "checkout" will actually work.

Ha, nice one, it seems like your trick would actually work:

https://github.com/open-vsx/publish-extensions/blob/0be92c462d77e884aae5f49af127831f5d471ae9/publish-extensions.js#L104-L106

The preferred, more "official" way to do these things is with a dedicated `"prepublish"` option:

https://github.com/open-vsx/publish-extensions/blob/0be92c462d77e884aae5f49af127831f5d471ae9/publish-extensions.js#L111-L113

However, what is the purpose of deleting `.yarnrc`? I guess that, if you want to delete it before the `yarn install` happens, the `"prepublish"` option would indeed come in too late.

> Is there any way to test this before merging?

Unfortunately, not really. You could approximate a publish run locally by keeping only your extension in `extensions.json` and running `node publish-extensions`, but this isn't ideal because it will fail on OpenVSX calls (needs a `OVSX_PAT` token) and will actually attempt to publish for real.

But I'm fine with merging untested PRs, and then dealing with any resulting CI failures myself (and if that becomes problematic, I'll be encouraged to implement a better way to test PRs).

Reply:
> However, what is the purpose of deleting `.yarnrc`?

`.yarnrc` from the repository contains this:

```
yarn-path ".yarn/releases/yarn-1.22.4.js"
```

which breaks everything as yarn itself is not included in the repository.

> I guess that, if you want to delete it before the `yarn install happens`, the `"prepublish"` option would indeed come in too late.

Right.

Reply:
Thought just a moment ago: maybe that file was pushed by mistake in the first place? I'll try to send pull request to the upstream.